perpendicular-flap/solid-fake/fake.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import division
+
+import numpy as np
+import precice
+
+
+def displace_flap(x, y, t, flap_tip_y):
+    x_displ = np.zeros_like(x)
+    y_displ = np.zeros_like(y)
+    # first, get displacement independent of x, only dependent on y and t
+    # maximal displacement at flap tip should be 0.5
+    # initially, flap's displacement is 0
+    x_displ = np.minimum(((np.sin(3 * np.pi * t + np.arcsin(-0.95)) + 0.95) / 8) * y / flap_tip_y, 0.5 * y / flap_tip_y)
+
+    displ = np.zeros((len(x), 2))
```

Comment:
```suggestion
    dimensions = 2
    displ = np.zeros((len(x), dimensions))
```
Again: Makes it more explicit that this only works in 2D.